### DISCUSSION TITLE
Problem with a background image component

### DISCUSSION THREAD

**Discussion by SaschaRadojkovic** [neuefische/questions#491] [Web] [answered]
I have found a background image component tutorial  here the link to this [github repository](https://github.com/TomDoesTech/perfect-background-image-tutorial). 

the tutorial is[ here](https://www.youtube.com/watch?v=IVfqp_EIuwc)

the "problem" is that i have to wrap every element and work with the z-index to make the elements visible and i thought about if this is a good way to go . I have done it yet but i am not shure if this is good practice and i need a professional opinion maybe to avoid this wrapping chaos. Also on one page i could not get the expected result of elements in front of the background image!

**Comment by doemser** [comment, marked as answer]
You could set the next image on a negative zIndex, like -1.

```javascript
<>
      <Image
        src="/magician.png"
        alt="Picture of a magician"
        layout="fill"
        objectFit="cover"
        objectPosition="center"
        style={{ zIndex: -1 }}
      />

     ..content on top..
       
</>

```

So you don't bend your whole project for a background image.

**Reply by SaschaRadojkovic** [reply]
it took nearly the hole day to get it work with  but it works now without wrapping Hell :)
